src/test/java/org/broadinstitute/hellbender/tools/spark/pathseq/PathSeqFilterSparkIntegrationTest.java

```diff
@@ -1,94 +1,99 @@
 package org.broadinstitute.hellbender.tools.spark.pathseq;
 
 import htsjdk.samtools.ValidationStringency;
+import htsjdk.samtools.metrics.MetricsFile;
 import org.broadinstitute.hellbender.CommandLineProgramTest;
 import org.broadinstitute.hellbender.utils.test.ArgumentsBuilder;
 import org.broadinstitute.hellbender.utils.test.SamAssertionUtils;
 import org.testng.Assert;
+import org.testng.annotations.DataProvider;
 import org.testng.annotations.Test;
 
 import java.io.File;
 
 public class PathSeqFilterSparkIntegrationTest extends CommandLineProgramTest {
 
+    static final String imagePath = "src/test/resources/" + PathSeqFilterSpark.class.getPackage().getName().replace(".", "/") + "/hg19mini.fasta.img";
+    static final String libraryPath = "src/test/resources/" + PathSeqFilterSpark.class.getPackage().getName().replace(".", "/") + "/hg19mini.hss";
+
+    @DataProvider(name = "pathseqFilterTestData")
+    public Object[][] getTestData() {
+        return new Object[][]{
+                {"basic_input.bam",
```

Comment:
Is this the default indentation in IntellJ, it does not look great.

Reply:
Yes it is. I removed one tab, which looks slightly better...